lib/spack/spack/test/sbang.py

```diff
@@ -52,7 +60,7 @@
 
 @pytest.fixture  # type: ignore[no-redef]
 def sbang_line():
-    yield '#!/bin/sh %s/bin/sbang\n' % spack.store.layout.root
+    yield '#!/bin/sh {0}{1}bin{1}sbang\n'.format(spack.store.layout.root, os.sep)
```

Comment:
Since this wouldn't apply on Windows (and because all these tests are skipped on windows) I think you could undo this change (although a comment about how the path format is OK because its on Linux/Mac OS could be useful)

Reply:
Agreed. Handled.